packages/react/src/Testimonial/Testimonial.tsx

```diff
@@ -111,12 +111,9 @@ const Root = forwardRef(_Root)
  * Testimonial quote child element
  * <Testimonial.Quote>
  */
-type QuoteProps = {
-  children: string | Array<string | React.ReactElement<HTMLElement>>
-} & React.HTMLAttributes<HTMLElement> &
-  BaseProps<HTMLElement>
+type QuoteProps = React.HTMLAttributes<HTMLElement> & BaseProps<HTMLElement>
 
-function _Quote({children, className}: QuoteProps, ref) {
+function _Quote({children, className}: PropsWithChildren<QuoteProps>, ref) {
   const childrenArray = useMemo(() => React.Children.toArray(children), [children])
```

Comment:
Minor thing, but you don't need `PropsWithChildren` here as that's included in `HTMLAttributes`. You could also use `HTMLQuoteElement` over `HTMLElement` to be a bit more explicit, although it makes no difference afaik

```suggestion
type QuoteProps = React.HTMLAttributes<HTMLQuoteElement> & BaseProps<HTMLElement>

function _Quote({children, className}: QuoteProps, ref) {
```

Reply:
Nice! Didn't know that extended DOMAttributes. Was mostly trying to get rid of the manual children declaration.